### DISCUSSION TITLE
插件分享

### DISCUSSION THREAD

**Discussion by read-cat** [read-cat/read-cat#2] [General]
### 该贴用于插件分享，请勿回复无关内容

> **若您想分享插件，请在此贴以如下格式回复即可**

类型：书源（插件类型，若是书城类插件则填写书城）
作者：XXX
描述：XXX

**Comment by moomewa** [comment]
### 该贴用于插件分享，请勿回复无关内容

> 若您想分享插件，请在此贴以如下格式回复即可

类型：书源
作者：moomewa
插件开发文档：https://docs.qq.com/doc/DQUZOQ3BLeWNrRFFp
123pan：https://www.123pan.com/s/OwzA-qQGsd.html

**Reply by powermill** [reply]
希望大佬能开发出一个  安卓  “阅读”app 的书源转换成  readcat 书源的 转换器，这样大家就不要纠结 readcat书源少的问题了。

**Reply by luoxda** [reply]
直接支持“阅读3.0”的书源更好吧？这个app是开源的，应该可以找到相关书源的处理代码。

**Comment by Passing-of-A-Dream** [comment]
类型：书源
作者：Passing-Dream
阿里云盘：https://www.alipan.com/s/UgEEtDjak15

**Reply by q7954** [reply]
网盘失效了

**Reply by Passing-of-A-Dream** [reply]
123盘最新的：https://www.123pan.com/s/7cOATd-ABOPA.html

**Reply by wang2425** [reply]
过期了

**Comment by iLEX520** [comment]
类型：书源
作者：群里各位大佬
文件：插件合集
123pan：https://www.123pan.com/s/xxsZjv-rvoUA.html

**Reply by iLEX520** [reply]
5月10日新增www.99mk.cc.ts.js

**Reply by iLEX520** [reply]
5月10日新增www.xuanyge.info.ts.js，感谢Demo大佬

**Reply by CeobeEX** [reply]
可以搜到书了，感谢。

**Reply by yxcqr** [reply]
感谢分享，能搜索到书源了，如果能像阅读APP那样，就可以有更多的源可用了，希望加入。

**Comment by bashimiao** [comment]
类型：书源
作者：云上
文件：插件合集
123pan：https://www.123pan.com/s/DnfUVv-dLuUH.html

**Reply by bashimiao** [reply]
6月12日1:59更新>天章小说https://www.tianz.net/

**Reply by bashimiao** [reply]
6月12日2:17更新>下书小说https://m.moshayuan.com/  书多速度较慢

**Reply by bashimiao** [reply]
有需要写的书源，请留言网址即可

**Reply by bashimiao** [reply]
6月12日3:33更新>笔趣阁-88yuedu http://88yuedu.com/ 书多

**Reply by yxcqr** [reply]
感谢老板分享，支持没得说。

**Reply by a809853681** [reply]
番茄原创的搜不到

**Reply by bashimiao** [reply]
后续会找找番茄源的书，因为番茄小说书名原因大多数源只能识别发布时的书名，有好 的书源可以分享

**Reply by yz0625** [reply]
大佬可以写这两个吗？>>>>卡夜阁：https://www.kygxs.com/   飞速中文：https://m.feisxs.com/

**Reply by bashimiao** [reply]
这两个不好写，有加密数据

**Reply by bashimiao** [reply]
6月20日2:16更新👇
斋书网>https://www.tianz.net
必去小说>http://www.biquw.la
九五读书>http://www.95dushu.net

**Reply by ssyangchn** [reply]
大佬求一个www.dushu.com插件

**Reply by Kenekolan** [reply]
大佬，可以弄下这个网站吗，久久小说网：https://jjjjxs.com/

**Reply by Anthrax7440** [reply]
大佬，弄下这个网站 https://www.kingho.org/ KING小说

**Reply by madisonkane** [reply]
大佬，可以帮忙做一下哔哩轻小说网站https://www.linovelib.com/的插件吗？编程小白真的看不懂手册讲解啊TAT

**Comment by starfire-hit** [comment]
类型：书源
作者：starfire-hit
文件：wenkuchina.com.js
夸克：https://pan.quark.cn/s/7941d33fd72d

**Reply by starfire-hit** [reply]
还没加群，自己用的；对搜索结果1条或多条时分别进小说信息页或显示搜索结果列表的两种情况做了处理；对章节内有分页做了处理；能正常朗读；只测了自己在看的几篇小说，没充分测试，先放在这了。

**Reply by starfire-hit** [reply]
这网站小说文本里html标记切分乱七八糟的，资源应该也是爬过来的

**Reply by starfire-hit** [reply]
更新了下搜索结果分页的拼接，https://pan.quark.cn/s/35d5c630c9ec

**Comment by Wang-YLong** [comment]
类型：书源
作者：doge
文件：<del>番茄小说0.0.2.js</del> 番茄小说0.0.3.js
123pan：https://www.123865.com/s/pZo0Vv-znGr

**Reply by Wang-YLong** [reply]
+  **妙书斋** 书源
+ **哔哩轻小说** 书源（无图）

**Reply by Wang-YLong** [reply]
- **哔哩轻小说** 书源（**有图**）
- **全本漫画网** 书源（漫画源）_demo 不再更新_

**Reply by moomewa** [reply]
建议将分享链接有效期设为永久

**Comment by lathert** [comment]
类型：书城
作者：doge
文件：书城&书源
123pan：
- https://www.123865.com/s/pZo0Vv-bsGr
- https://www.123865.com/s/pZo0Vv-xOGr

**Reply by mlfhls** [reply]
番茄书源，只能用id和链接搜索吗，为什么不能用书名

**Reply by lathert** [reply]
一开始是可以的，后来番茄改了加密方法，能力有限，没看懂🤣

**Comment by qq173972819** [comment]
类型：书源
作者：Insane
文件：笔趣阁3.js
123pan：https://www.123684.com/s/IqF5Vv-LPkod

**Comment by qq173972819** [comment]
类型：书源
作者：Insane
文件：看点中文网.js
123pan：https://www.123684.com/s/IqF5Vv-3Xkod

**Comment by qq173972819** [comment]
类型：书源
作者：Insane
文件：七星阁.js
123pan：https://www.123684.com/s/IqF5Vv-PXkod?提取码:4vGy

**Reply by qq173972819** [reply]
https://www.123684.com/s/IqF5Vv-XXkod

**Comment by moomewa** [comment]
类型：TTS离线引擎
作者：moomewa
123pan：https://www.123684.com/s/OwzA-3mnsd
夸克网盘：https://pan.quark.cn/s/e808d5495fc3
描述：该离线引擎为网易开源的TTS引擎[**EmotiVoice**](https://github.com/netease-youdao/EmotiVoice)
音色较好，但配置复杂

**Reply by qq173972819** [reply]
可以换个网盘么   太大了 下不了

**Comment by qq173972819** [comment]
类型：书源
作者：Insane
文件：新百强小说.js
123pan：https://www.123684.com/s/IqF5Vv-WXkod

**Comment by qq173972819** [comment]
类型：书源
作者：Insane
文件：飞速中文网.js
123pan：https://www.123684.com/s/IqF5Vv-9nkod

**Comment by qq173972819** [comment]
类型：书源书城
作者：Insane
文件：手机文学网
123pan：https://www.123684.com/s/IqF5Vv-2nkod

**Reply by qq173972819** [reply]
书城固定加载300条数据，加载会慢一些。      希望软件在后期版本能加上分页多次插入数据         一股脑把数据全丢过去需要加载时间

**Comment by tianjiaHome** [comment]
类型：书源书城
作者：SuperJia
文件：四库书屋
通过网盘分享的文件：4kswsc.ts
链接: https://pan.baidu.com/s/1Rb_vihPs0Oh1q-7sQ5XMXQ?pwd=vxxn 提取码: vxxn

**Reply by AlphaCatMeow** [reply]
请教一下，为什么是ts类型的文件，不是应该是js格式吗？

**Reply by xiaochen296** [reply]
> 请教一下，为什么是ts类型的文件，不是应该是js格式吗？

ts要自己编译，很简单，看看插件教程就会了

**Comment by AlphaCatMeow** [comment]
类型：书源
作者：本页面各位大佬
文件：书源合集
123盘：https://www.123684.com/s/XASDVv-Y1cqh

**Reply by chenliuyong** [reply]
感谢课代表！

**Comment by qq173972819** [comment]
类型：书源
作者：Insane
文件：情豆书坊
123盘：https://www.123684.com/s/IqF5Vv-XKkod

**Reply by yeyuyayi** [reply]
大佬能把这几个网站也做成书源吗？https://www.jpxs123.cc/tongren/index.html
https://www.qbtr.cc/tongren/index.html
https://www.ffxs8.com/trxs/index.html
https://www.tongrenquan.org/tongren/index.html
https://www.bixiange.top/trxs/index.html
https://www.trxs.me/tongren/index.html
https://www.trxs.cc/tongren/index.html

**Comment by zhangsans1** [comment]
向大佬们请教，我在使用自带的Edge TTS Engine 插件朗读小说时出现这个错误无法使用：Unexpected server response: 403  碰到这种情况应该怎么办？如何想用本地TTS离线引擎具体应该怎么操作？

**Reply by fyxsky** [reply]
ReadCat 不会调用你本机的 Edge 浏览器进程。

它的 “Edge TTS” 本质是请求微软在线 TTS 接口（speech.platform.bing.com），只是请求头会伪装成 Edge 客户端，所以需要一个“Edge 版本号”字符串用于构造 UA/协议参数。

所以在 Mac 没装 Edge 也能用。
你可以：
直接改成较新的 143.0.3650.75 即可。
不过注意看我回复的时间。如果你当前的时间和我回复的时间差距已经很远。那么，安装一个 Edge，看一下当前的版本，填进去就行了。

**Comment by qq173972819** [comment]
最近Edge TTS被墙了  用梯子要开tun模式  会对别的请求造成影响故又研究了另一个方法；
手机或者虚拟机安装MultiTTS   打开转发功能   打开电池白名单、后台避免被杀后台； 
MultiTT  APK  离线语音包  下载地址：https://www.123684.com/s/IqF5Vv-x6kod
TTS插件  https://www.123684.com/s/IqF5Vv-f6kod
导入插件点击设置  输入手机或者虚拟机的ip  转发端口默认不修改
虚拟机需要在网络设置打开桥接 安装桥接网卡

**Reply by catvox** [reply]
用singbox设置规则再开tun就好了.

**Reply by qq173972819** [reply]
有人没有梯子，可以用手机转发电脑听
